lib/fcrepo_wrapper/configuration.rb

```diff
@@ -132,8 +132,16 @@ def default_configuration_paths
         ['.fcrepo_wrapper', '~/.fcrepo_wrapper']
       end
 
+      def default_download_dir
+        if defined? Rails
+          Rails.root.to_s + '/tmp'
```

Comment:
Could we do `File.join(Rails.root, 'tmp')` to maybe play nice with windows users?